rgw/v2/tests/s3_swift/test_data_omap_offload.py

```diff
@@ -0,0 +1,214 @@
+"""
+test_data_omap_offload - Test data omap offload 
+Usage: test_data_omap_offload.py -c <input_yaml>
+<input_yaml>
+    Note: any one of these yamls can be used
+	test_data_omap_offload.yaml 
+	test_data_omap_offload_change_datatype_to_omap.yaml
+	test_data_omap_offload_change_datatype_to_fifo.yaml
+	test_data_omap_offload_multipart.yaml 
+	test_data_omap_offload_versioned_bucket.yaml
+Operation:
+    with default datalog_backing verify 
+	change the default datalog_backing and verify [applicable to nautilus]
+"""
+
+import os
+import sys
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+import argparse
+import hashlib
+import json
+import logging
+import time
+import traceback
+
+import v2.lib.manage_data as manage_data
+import v2.lib.resource_op as s3lib
+import v2.utils.utils as utils
+from v2.lib.exceptions import RGWBaseException, TestExecError
+from v2.lib.resource_op import Config
+from v2.lib.rgw_config_opts import CephConfOp, ConfigOpts
+from v2.lib.s3.auth import Auth
+from v2.lib.s3.write_io_info import BasicIOInfoStructure, BucketIoInfo, IOInfoInitialize
+from v2.tests.s3_swift import reusable
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+from v2.utils.utils import RGWService
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def test_exec(config):
+
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    write_bucket_io_info = BucketIoInfo()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    ceph_conf = CephConfOp()
+    rgw_service = RGWService()
+
+    # check the default data log backing
+    default_data_log = reusable.get_default_datalog_type()
+    if "omap" in default_data_log:
+        log.info("omap is the default data log backing in nautilus")
+    else:
+        log.info("fifo is the default data log backing in pacific and above")
+
+    # check sync status if a multisite cluster
+    reusable.check_sync_status()
+
+    # create user
+    all_users_info = s3lib.create_users(config.user_count)
+    for each_user in all_users_info:
+        # authenticate
+        auth = Auth(each_user, ssl=config.ssl)
+        if config.use_aws4 is True:
+            rgw_conn = auth.do_auth(**{"signature_version": "s3v4"})
+        else:
+            rgw_conn = auth.do_auth()
+        objects_created_list = []
+
+        # change the default datalog backing to FIFO
+        if config.test_ops["change_datalog_backing"] == "fifo":
+            log.info("change the default logtype from omap to fifo")
+            cmd = "radosgw-admin datalog type --log-type=fifo"
+        else:
+            log.info("change the default logtype from fifo to omap")
+            cmd = "radosgw-admin datalog type --log-type=omap"
```

Comment:
you could use one singe `log.info` here which can come before

Reply:
done! modified the usage